graph/src/schema/input_schema.rs

```diff
@@ -395,6 +435,332 @@ impl EnumMap {
     }
 }
 
+#[derive(PartialEq, Debug, Clone)]
+pub enum AggregateFn {
+    Sum,
+    Max,
+    Min,
+    Cnt,
```

Comment:
I think we don't lose much by calling this Count

Reply:
Heh, yeah, not sure why I wrote that. Changed it